index.bs

```diff
@@ -187,29 +187,95 @@ An orchestrator MUST respond to the following three types of triggers:
 - an observed state changes to watched [=data pod=] resources
 - a scheduled trigger from the internal time-based event scheduler
 
-## Receving incoming Linked Data Notification 
+## Receving Linked Data Notifications ## {#trigger-ldn}
+An Orchestrator MAY accept [Linked Data Notifications (LDN)](https://www.w3.org/TR/ldn/) as triggers for policy actions.
+In this case, the Orchestrator MUST be a compliant [[LDN]] <a href="https://www.w3.org/TR/ldn/#receiver">Receiver</a>. 
```

Comment:
Why the switch to receiver? Making the Orchestrator a LDN consumer and not a receiver was a deliberate choice, else the orchestrator has to be an (LDP) server that is able to process GET and POST request to receive notifications directly.